pkg/cmd/cli/cmd/edit.go

```diff
@@ -417,7 +417,7 @@ func applyPatch(delta *jsonmerge.Delta, info *resource.Info, version string) err
 // their updates were preserved.
 func preservedFile(err error, path string, out io.Writer) error {
 	if len(path) > 0 {
-		if _, err := os.Stat(path); !os.IsNotExist(err) {
+		if _, err = os.Stat(path); !os.IsNotExist(err) {
```

Comment:
this sort of thing is why this PR should be reviewed slowly and carefully... I'd need to dig into what the expected behavior is